.gitignore

```diff
@@ -0,0 +1,13 @@
+*.iml
```

Comment:
please update this `.gitignore` as well

Reply:
Check this SO question and the accepted answer:
https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore